GA/config.py

```
@@ -0,0 +1,307 @@
+    filenames = [R00_fpath, R03_fpath, R05_fpath, R07_fpath, R10_fpath, Hstrut_fpath, Dstrut_fpath]
+
+    # First check if all files are present
+    missing_files = [f for f in filenames if not f.exists()]
+    if missing_files:
+        raise FileNotFoundError(f"Missing files: {', '.join(map(str, missing_files))}")
+
+    # Compute parameterization for the airfoil section at r=0R
+    R00_section = AirfoilParameterization().FindInitialParameterization(reference_file=R00_fpath,
+                                                                        plot=False)
+    # Compute parameterization for the airfoil section at r=0.3R
+    R03_section = AirfoilParameterization().FindInitialParameterization(reference_file=R03_fpath,
```

Ruff (0.8.2)

Local variable `R03_section` is assigned to but never used

Remove assignment to unused variable `R03_section`

(F841)